templates/style.css

```diff
@@ -0,0 +1,162 @@
+    border-radius: 35px 0 35px 0;
+}
+
+p{
+    color:chocolate;
+    position:relative;
+    background: var(--p-gradient);
+    margin-left: 75px;
+    margin-right: -40px;
+    padding: 20px;
+    border-bottom: var(--p-border) solid;
+    border-width: 0 2 2 0;
+}
+p::before{
+    content: '';
+    background: orange;
+    position:absolute;
+    left: -20px;
+    top: 0px;
+    display: block;
+    width: 20px;
+    height: 100%;
+    border-bottom: 3px var(--p-border) solid;
+}
+small{
+    color: aliceblue;
+    margin-left: 100px;
+    margin-right: -40px;
+    background: var(--p-gradient);
+    padding: 10px;
+    display: block;
+    border-bottom: var(--p-border) solid;
+}
+h2{
+    color: brown;
+    text-align: center;
+    text-decoration-line: underline;
+}
+div{
+    background-color: rgba(241, 241, 241, 0.6);
+    line-height: 40px;
+    
+    margin-left: 40px;
+    margin-right: 40px;
+    text-align:center;
+    padding: 10px;
+    border: rgba(255, 255, 255, 0.685) 2px solid;
+    border-radius: 50px;
+    
+}
+div > a{
+    position: relative;
+    counter-increment: hue;
+    display: inline-block;
+    background: transparent;
+    
+    border-radius: 25px;
+    padding: 5px;
+    line-height: 20px;
+    text-decoration-line: none;
+    box-shadow: 2px 5px 5px rgba(255, 0, 221, 0.226);
+    color: darkred;
+    overflow: hidden;
+}
+div > a::before{
+    content: '';
+    position:absolute;
+    width: 200%;
+    height: 200%;
+    left: -50%;
+    top: -50%;
+    background-color: red;
+    z-index: -2;
+}
+div > a::after{
+    background-color:rgb(255, 255, 255);
+    content: '';
+    position: absolute;
+    display: inline-block;
+    left: 2px;
+    top: 2px;
+    width: calc(100% - 4px);
+    height: calc(100% - 4px);
+    border-radius: 25px;
+    inset: 2px;
+    z-index: -1;
+    size: 1;
+    transition: filter 1s;
+}
+div > a:hover::before{
+    content: '';
+    position:absolute;
+    width: 200%;
+    height: 1000px;
+    top: calc(-500px + 50%);
+    left: -50%;
+    background-image: conic-gradient(hotpink,red,orange,yellow,lime,turquoise,indigo,violet);
+    transform-origin: center;
+    animation: animate linear infinite;
+    animation-duration: 4s;
+    z-index: -2;
+}
+
+h1::before{
+    content: '';
+    position: fixed;
+    display: block;
+    width:200px;
+    height:200px;
+    left: 50%;
+    top: 50%;
+    border-radius: 50%;
+    background-color: rgb(0, 225, 255);
+
+    animation: animate 20s linear infinite;
+    filter: blur(100px);
+    z-index: -1;
+
+    box-shadow: -250px -250px hotpink, -250px 0px red, -250px 250px orange, 0px -250px yellow, 0px 250px lime, 250px -250px turquoise, 250px 0px indigo, 250px 250px violet;
+    opacity: 40%;
+    scale: 2;
+}
+
+@keyframes animate
+{
+    0%{
+        rotate: 0deg;
+    }
+    50%{
+        rotate: 180deg;
+    }
+    100%{
+        rotate: 360deg;
+    }
+    
+}
+
+p, div, small{
+    backdrop-filter: blur(25px);
+}
```

Comment:
Instead of creating a new file, please replace the existing `static/style.css` file. That's the one actually used by the final site.